examples/canvas/src/lib.rs

```diff
@@ -72,19 +83,33 @@ fn update(msg: Msg, model: &mut Model, orders: &mut impl Orders<Msg>) {
                 Color::A
             };
         }
+        Msg::Zoom(wheel_ev) => {
+            if wheel_ev.delta_y() < 0. {
+                model.zoom -= 0.1;
+            } else if wheel_ev.delta_y() > 0. {
+                model.zoom += 0.1;
+            }
+        }
```

Comment:
If we update `wheel_ev` (see the comment below), then we can rewrite it to:
```suggestion
        Msg::WheelDeltaY(delta_y) => {
            model.zoom += if delta_y < 0. { -0.1  } else { 0.1 };
        }
```